scripts/build.js

```diff
@@ -4,14 +4,16 @@ const glob = require("glob")
 const fs = require("fs-extra")
 const path = require("path")
 const write = require("./write")
+const writeNative = require("./writeNative")
 
 if (!fs.existsSync(".build/svg")) {
   console.error("Requires optimization step to be run first")
   process.exit(1)
 }
 
 const filepaths = glob.sync(".build/svg/*.svg")
-const BUILD_PATH = path.join(__dirname, "..", ".build", "src")
+const BUILD_PATH = path.join(__dirname, "..", ".build", "src", "web")
+const BUILD_PATH_NATIVE = path.join(__dirname, "..", ".build", "src", "native")
```

Comment:
I had to move the web svg components to `web` so the native components doesn't get auto-completed by the IDEs to `@artsy/icons/native/XIcon` (due to the `./*` exports in `package.json`) because the react native metro bundler doesn't resolve from "exports", but resolves from  "react-native" (as far as I know and tested)